test/stdlib/SmallString.swift

```diff
@@ -1,4 +1,6 @@
-// RUN: %target-build-swift -Xfrontend -disable-access-control -module-name a %s -o %t.out -O
+// RUN: %empty-directory(%t)
+// RUN: %gyb %s > %t/main.swift
```

Comment:
This test isn't a gyb file.

Reply:
Ah, forgot to change that back to cp after I figured out that the part I was missing was the empty-directory line.